packages/bpk-component-scrollable-calendar/src/bpk-scrollable-calendar.scss

```diff
@@ -20,4 +20,9 @@
 
 .bpk-scrollable-calendar {
   display: flex;
+  flex-direction: column;
```

Comment:
This has been renamed but no reference to it has been updated ... was this a phantom file? 😱

Reply:
Yep. We snuck it past review last time somehow. :)